src/runtime-rs/Makefile

```diff
@@ -8,14 +8,250 @@
 # Use the following format:
 # '##VAR VARIABLE_NAME: help about variable'
 # '##TARGET TARGET_NAME: help about target'
-
+PROJECT_TYPE = kata
 PROJECT_NAME = Kata Containers
+PROJECT_TAG = kata-containers
 PROJECT_URL = https://github.com/kata-containers
 PROJECT_COMPONENT = containerd-shim-kata-v2
 CONTAINERD_RUNTIME_NAME = io.containerd.kata.v2
 
+#Get ARCH.
+ifeq ($(ARCH),)
```

Comment:
Since dragonball is rust, please could you consider including [`utils.mk`](https://github.com/kata-containers/kata-containers/blob/main/utils.mk) to avoid duplication and ensure we use a consistent install process.

Reply:
fixed